docs/src/use/configure/configuration-files.md

```diff
@@ -342,6 +343,31 @@ Here, the `js.configs.recommended` predefined configuration is applied first and
 
 For more information on how to combine predefined configs with your preferences, please see [Combine Configs](combine-configs).
 
+### Configuration Naming Conventions
+
+The `name` property is optional, but it is recommended to provide a name for each configuration object, especially when you are creating shared configurations. The name is used in error messages and the config inspector to help identify which configuration object is being used.
+
+The name should be descriptive of the configuration object's purpose and scoped with the configurations name or plugin name. Using `/` as the seperator. For example, if you are creating a configuration object for a plugin named `eslint-plugin-example`, you might add `name` to the configuration objects with the `example/` prefix:
+
+```js
+export default {
+    configs: {
+        recommended: {
+            name: "example/recommended",
+            rules: {
+                "no-unused-vars": "warn"
+            }
+        },
```

Comment:
What would be the convention if the configuration is an array of configuration objects? Would it be the same name for all configuration objects?

Reply:
Good point! I mentioned it here 8cccc58d91f14cd50e479dcc6ea4c99620f5f96f

Supposely the config inspector could have an indicator of duplicated names to help with that as well.